package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "opensea-js",
-  "version": "6.0.1",
+  "version": "6.0.2",
```

Comment:
ah didn't realize we already did 6.0.1, can you make a smol gh release for it? Ideally you can just use the npm publish workflow so you make the release to publish on npm automatically